src/v/cluster/tests/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ rp_test(
   BINARY_NAME test_cluster
   SOURCES ${srcs}
   LIBRARIES v::seastar_testing_main v::application v::storage_test_utils v::v8_engine
-  LABELS cluster
+  LABELS cluster disable_on_ci  # Disabled for https://github.com/vectorizedio/redpanda/issues/2175
```

Comment:
This is a huge test. Can just the requisite source file be removed?

Reply:
PSA: `bin/test_cluster_rpunit --list_content` - which is what I want the CTest driver to use.